hack/.golint_failures

```diff
@@ -350,6 +350,7 @@ pkg/util/labels
 pkg/util/mount
 pkg/util/netsh/testing
 pkg/util/node
+pkg/util/normalizer
```

Comment:
This shouldn't fail golint. Either create an issue to follow up or fix

Reply:
Unfortunately this package implement an interface defined in a vendored package, that isn't go lint compliant. So the the package `pkg/util/normalizer` will never pass golint verify

Reply:
ok, thanks for the explanation